### DISCUSSION TITLE
네이버 TV

### DISCUSSION THREAD

**Discussion by mildtwo** [List-KR/community#50] [Q&A]
안녕하세요 개발자님
그리고 들러주시는 개발에 도움주시는 유저님들
덕분에 국내에서 쾌적하게 컴 쓰고 있습니다 넘 감사합니다

제가 예전부터 PC에서 네이버 스포츠 라이브가 재생이 안돼서요
네이버tv 포함하여 네이버 다른 동영상들은 라이브까지 모두 문제없이 재생되는데요  
'스포츠'에서 만큼은 라이브가 재생이 안됩니다
스포츠에서 다른 영상은 또 재생됩니다
재생시 설치한 네이버 라이브 스트리밍 커넥터의 문제 때문일까요?
몇년전부터 그랬고 adguard에 메일도 몇번 넣어봤는데 여태 이렇게 쓰고 있습니다.

적용중인 필터는
adguard 베이스필터
adguard url 추적필터
adguard 추적보호필터
adguard 방해요소필터
adblock warning removal list
fanboy's annoyances
쿠키를 신경쓰지 않습니다
online malicious url blocklist
nocoin filter list
lisk-kr
yourslist

확장프로그램은
adguard 어시스턴스
adguard extra
adguard 팝업차단기
micro shield
web of trust

이렇게 쓰고있고

제가 따로 정의한 필터링 규칙은 아래와 같습니다

@@||tinyurl.is^$important,document
@@||www.instagram.com^$important,document
@@||auto.danawa.com^$important,document
@@||www.gmarket.co.kr^$important,document
||mud-kage.kakao.co.kr/dn^$app=KakaoTalk.exe
namu.wiki###xeZKSV0Ou > div.\37 511ff4e:nth-child(2) > div.\37 511ff4e > div.\37 511ff4e > div.\37 511ff4e > div.\37 511ff4e > div.\37 511ff4e > div.\37 511ff4e > div.\37 511ff4e > div.\37 511ff4e > div.rV9gy7iH > div.k6yEBAr-:first-child > div:nth-child(7) > div > div > div > div > div > div:nth-child(6) > div > div > div > div > div > div > div.\33 6e95048:last-child > div.\33 6e95048 > div.\33 6e95048 > div.\33 6e95048 > div.\33 6e95048 > div.\33 6e95048 > div:first-child > div._1\+UTAYHc > div:last-child > div > table.NvgE49QI._50c31d28d8025ec62943a1dd0c93b9e0 > tbody
@@||www.dhlottery.co.kr^$important,document
youtube.com###hover-area
@@||me2disk.com^$important,document
@@||www.giro.or.kr^$important,document
wjdtmddnjs6788.tistory.com###jb-page > div.jb-background.jb-background-main:nth-child(4) > div.jb-container.jb-container-main > div.jb-row.jb-row-main:nth-child(2) > div.jb-column.jb-column-content:first-child > div.jb-cell.jb-cell-content.jb-cell-content-article > article > section:nth-child(5) > div.jb-content.jb-content-article > div.jb-article:last-child > div.revenue_unit_wrap:nth-child(5)
wjdtmddnjs6788.tistory.com###jb-page > div.jb-background.jb-background-main:nth-child(4) > div.jb-container.jb-container-main > div.jb-row.jb-row-main:nth-child(2) > div.jb-column.jb-column-content:first-child > div.jb-cell.jb-cell-content.jb-cell-content-article > article > section:nth-child(5) > div.jb-content.jb-content-article > div.jb-article:last-child > div.tt_article_useless_p_margin.contents_style:nth-child(4) > div.revenue_unit_wrap:nth-child(63)
@@||best.aliexpress.com^$important,document
@@||devuploads.com^$important,document
@@||banking.nonghyup.com^$important,document

이상입니다 항상 감사합니다 개발자님 유저님들

**Comment by piquark6046** [comment]
일단, AhnLab 안티-바이러스가 설치되어 있나요?

**Reply by piquark6046** [reply]
@mildtwo

**Reply by mildtwo** [reply]
신경써주셔서 감사합니다
아니요 설치 안했고 은행사이트 들어갈 때 깔리는 ahnlab safe transaction 은 가끔 켜지는데 요새 모두의플러그 앱을 알게 되어서 윈도 부팅하고 바로 끕니다
특히 윈도 재설치해도 같습니다

**Reply by piquark6046** [reply]
사용자 규칙을 모두 비활성화하시고 아래 필터만 활성화하시면 후에 새로고침해보세요:
- adguard 베이스필터
- adguard url 추적필터
- adguard 추적보호필터
- adguard 방해요소필터
- lisk-kr

**Reply by piquark6046** [reply]
그래도 안되면 설치된 프로그램을 모두 말씀해주세요.

**Reply by mildtwo** [reply]
말씀해주신 사용자 규칙 설정 방법으로 해봤지만 해결이 안되어 여러가지 먼저 체크해보다가
일단 제 경우에서 될 때와 안될 때 차이를 알아냈습니다.
네트워크 설정 - 트래픽 필터링 - 트래픽 필터링 활성화 [체크] -  WFP 네트워크 드라이버 사용 [체크 해제] 까지 되어 있을 때 네이버 스포츠 잘 됩니다
다만 , WFP 네트워크 드라이버 사용 - [체크] 되어 있으면 먼저 말씀드린 대로 다른 문제는 없이 네이버 스포츠 영상만 재생이 안됩니다. 
문제는 WFP 네트워크 드라이버 사용을 체크 해제하면 다른 응용프로그램에서 필터링이 안되는 경우가 너무 많은데 일반적으로 그렇나요?
확인이 쉬운 프로그램으로 예를 들면, 마이크로소프트 스토어에서 다운받는 게임들, 지뢰찾기 (Microsoft Minesweeper) 같은거는 대체로 광고 차단이 안되네요.
혹시 방법이 있을까요 선생님?

**Reply by piquark6046** [reply]
AhnLab 에서 배포하는 모든 안티-바이러스와 금융 프로그램을 모두 제거하신 후에 재부팅해보세요.
그 다음에, AdGuard for Windows를 재설치해보세요.
